projects/worker-azure-ai/actions/worker-azure-ocr/index.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const { worker, GenericError } = require("@adobe/asset-compute-sdk");
+const fetch = require("node-fetch");
+const fs = require('fs').promises;
+
+const DEFAULT_LANGUAGE = "en";
+
+exports.main = worker(async (source, rendition, params) => {
+    // get Azure credentials for API
+    let subscriptionKey = params && params.AZURE_OCP_KEY;
+    let endpoint = params && params.AZURE_OCP_ENDPOINT;
+
+    // WORKER_TEST_MODE is true when you are running worker unit
+    // tests using the `test-worker` command
+    // Dummy authorization parameters needed for mocks 
+    if (process.env.WORKER_TEST_MODE) {
+        subscriptionKey = "test-azure-key";
+        endpoint = "https://westus.api.cognitive.microsoft.com/";
+    }
```

Comment:
I would suggest `if/else` here with below condition. I nearly missed the below condition and asked to check it after assignments! ^^'

Reply:
I mean its not really an if/else situation. this is a separate check (and could even be put in a separate function as Mark did in his custom worker)

Reply:
to simplify the code, I just removed the credentials check. We can let the api call handle the error.

Reply:
For the record: I hope credentials you set for tests do not need verification, so it is an if/else situation.